test/compilable/test19656.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+
+TEST_DIR=${OUTPUT_BASE}
+
+mkdir -p ${TEST_DIR}
```

Comment:
Note that
- (1) the files should have been not generated on the fly and placed e.g. in `EXTRA_FILES`
- (2) `${OUTPUT_BASE}a` would have worked too

https://github.com/dlang/dmd/blob/4edae68767fcfd9b12ea4b638cb68e181dc0de49/test/tools/exported_vars.sh#L7